locations/philmel.yml

```diff
@@ -9,166 +9,231 @@ community: true
-
-  # northeast mesh 5GHz ac
   - vid: 10
     role: mesh
-    name: mesh_no_5ghz         # Peers: kiehlufer-core
-    prefix: 10.230.2.17/32
-    ipv6_subprefix: -1
-    mesh_metric: 1024
+    name: mesh_klunker
+    prefix: 10.230.2.32/32
+    ipv6_subprefix: -10
+    ptp: true
+    mesh_metric: 256
 
-  # northwest mesh 5GHz ac
   - vid: 11
     role: mesh
-    name: mesh_nw_5ghz          # Peers: liegewiese, sgfrd-core
-    prefix: 10.230.2.18/32
-    ipv6_subprefix: -2
+    name: mesh_ak36
+    prefix: 10.230.2.33/32
+    ipv6_subprefix: -11
+    ptp: true
     mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.5']
 
   - vid: 12
     role: mesh
-    name: mesh_ost            # Peers: Area51, delbrueck66
-    prefix: 10.230.2.19/32
-    ipv6_subprefix: -3
-    mesh_metric: 1024
+    name: mesh_nw_60ghz
+    prefix: 10.230.2.34/32
+    ipv6_subprefix: -12
 
+  # northeast mesh 5GHz ac
   - vid: 13
     role: mesh
-    name: mesh_vaterhaus
-    prefix: 10.230.2.20/32
-    ipv6_subprefix: -4
-    mesh_metric: 1024
+    name: mesh_no_5ghz
+    prefix: 10.230.2.35/32
+    ipv6_subprefix: -13
 
+  # northwest mesh 5GHz ac
   - vid: 14
     role: mesh
-    name: mesh_sued           # Peers: kranold18, GSBS2
-    prefix: 10.230.2.21/32
-    ipv6_subprefix: -5
-    mesh_metric: 1024
+    name: mesh_nw_5ghz
+    prefix: 10.230.2.36/32
+    ipv6_subprefix: -14
 
   - vid: 15
     role: mesh
-    name: mesh_sued_5ghz
-    prefix: 10.230.2.22/32
-    ipv6_subprefix: -6
-    mesh_metric: 1024
+    name: mesh_o_5ghz
+    prefix: 10.230.2.37/32
+    ipv6_subprefix: -15
 
   - vid: 16
     role: mesh
-    name: mesh_west           # Peers: emser97
-    prefix: 10.230.2.23/32
-    ipv6_subprefix: -7
-    mesh_metric: 1024
+    name: mesh_s_5ghz
+    prefix: 10.230.2.38/32
+    ipv6_subprefix: -16
 
   - vid: 17
     role: mesh
-    name: mesh_ak36
-    prefix: 10.230.2.24/32
-    ipv6_subprefix: -8
-    ptp: true
-    mesh_metric: 1024
-    mesh_metric_lqm: ['default 0.3']  # prefer klunker link
+    name: mesh_w_5ghz
+    prefix: 10.230.2.39/32
+    ipv6_subprefix: -17
 
-  - vid: 18
+  - vid: 20
     role: mesh
-    name: mesh_klunker
-    prefix: 10.230.2.25/32
-    ipv6_subprefix: -9
-    ptp: true
-    mesh_metric: 128
-
-  - vid: 19
+    name: mesh_nf_o_5
+    prefix: 10.230.2.40/32
+    ipv6_subprefix: -20
+    mesh_ap: philmel-nf-o-5ghz
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  - vid: 21
     role: mesh
-    name: mesh_nw_60ghz
-    prefix: 10.230.2.26/32
-    ipv6_subprefix: -10
-    ptp: true
-    mesh_metric: 1024
+    name: mesh_nf_s_5
+    prefix: 10.230.2.41/32
+    ipv6_subprefix: -21
+    mesh_ap: philmel-nf-s-5ghz
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  # - vid: 22
+  #   role: mesh
+  #   name: mesh_nf_w_5
+  #   prefix: 10.230.2.42/32
+  #   ipv6_subprefix: -22
+  #   mesh_ap: philmel-nf-w-5ghz
+  #   mesh_radio: 11a_standard
+  #   mesh_iface: mesh
+
+  - vid: 40
+    role: dhcp
+    prefix: 10.230.2.32/28
```

Comment:
These ips were already used for meshing. This should match the range in comment Line 108.